src/server/game/Maps/AreaDefines.h

```diff
@@ -96,6 +96,7 @@ enum AreaTableIDs : uint32
     AREA_EVERSONG_WOODS                                      = 3430,
     AREA_GHOSTLANDS                                          = 3433,
     AREA_HELLFIRE_PENINSULA                                  = 3483,
+    AREA_SILVERMOON_CITY                                     = 3487;
```

Comment:
```suggestion
    AREA_SILVERMOON_CITY                                     = 3487,
```

I don't mind adding entries to this if someone feels they need them